board/microchip/polarberry/post-image.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+#Create FAT linux Image (linux kernel  + rootfs + DTS)
+cp -R output/images ${BR2_EXTERNAL_MCHP_PATH}/board/microchip/polarberry
+/usr/bin/mkimage -A riscv -O linux -T flat_dt -f ${BR2_EXTERNAL_MCHP_PATH}/board/microchip/polarberry/fitimage_polarberry.its output/images/fitimage_polarberry.itb
```

Comment:
This (mkimage) should be ran in the binaries directory (output/images/). You can copy the .its to this directory. Don't forget to update the paths in your .its.